assets/src/containers/AssignmentPlanning.js

```diff
@@ -72,27 +72,24 @@ function AssignmentPlanning (props) {
         <Grid item xs={12}>
           <Paper className={classes.paper}>
             <Typography variant='h5' gutterBottom >Assignment Planning</Typography >
-            {assignmentData
-              ? <>
-                <FormControl className={classes.formControl}>
-                  <InputLabel>Courses</InputLabel>
-                  <Select
-                    value={assignmentFilter}
-                    onChange={event => setAssignmentFilter(event.target.value)}
-                  >
-                    <MenuItem value={0}>0% (all)</MenuItem>
-                    <MenuItem value={2}>2%</MenuItem>
-                    <MenuItem value={5}>5%</MenuItem>
-                    <MenuItem value={10}>10%</MenuItem>
-                    <MenuItem value={20}>20%</MenuItem>
-                    <MenuItem value={50}>50%</MenuItem>
-                    <MenuItem value={75}>75%</MenuItem>
-                  </Select>
-                </FormControl>
-                {AssignmentTable(assignmentData.plan)}
-              </>
-              : <Spinner />
-            }
+            <>
```

Comment:
Agree with this change. But just wonder why in the previous code the `<FormControl>` was also included in the chunk after the data is fetched.

Reply:
It's to make sure the dropdown selector doesn't disappear while the new data is being fetched.